src/components/News/NewsSmallCard.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import cx from 'classnames'
+import { dateDifferenceInWords } from '../../utils/dates'
+import MultilineText from '../MultilineText/MultilineText'
+import styles from './NewsSmallCard.module.scss'
+
+const NewsSmallCard = ({ title, description, sourceName, url, datetime }) => {
+  return (
+    <a
+      href={url}
+      target='_blank'
+      rel='noopener noreferrer'
+      className={cx(styles.link)}
+    >
+      <div className={styles.news}>
```

Comment:
`a` could be a single top-wrapper (without the `div` on LOC `15`), which will help us to avoid extra node creations